src/components/cards/TradeCardGP/TradeCardGP.vue

```diff
@@ -119,34 +126,41 @@ export default defineComponent({
   },
 
   setup() {
-    const highPiAccepted = ref(false);
-    const priceImpact = ref(0);
-    const isSell = ref(true);
-    const feeQuote = ref<FeeInformation | null>(null);
-    const feeExceedsPrice = ref(false);
-    const orderId = ref('');
-    const orderMetadata = ref<OrderMetaData | null>(null);
+    // COMPOSABLES
     const store = useStore();
     const router = useRouter();
     const auth = useAuth();
-    const { account, blockNumber } = useWeb3();
-
+    const { account, blockNumber, explorer } = useWeb3();
+    const { txListener } = useNotify();
     const { t } = useI18n();
+    const { gnosisOperator, gnosisExplorer } = useGnosisProtocol();
 
-    const getTokens = (params = {}) =>
-      store.getters['registry/getTokens'](params);
-    const getConfig = () => store.getters['web3/getConfig']();
-    const tokens = computed(() => getTokens({ includeEther: true }));
-
+    // DATA
+    const exactIn = ref(true);
+    const feeQuote = ref<FeeInformation | null>(null);
+    const feeExceedsPrice = ref(false);
+    const orderId = ref('');
+    const orderMetadata = ref<OrderMetaData | null>(null);
+    const updatingQuotes = ref(false);
+    const latestTxHash = ref('');
+    const txHash = ref('');
     const tokenInAddress = ref('');
     const tokenInAmount = ref('');
     const tokenOutAddress = ref('');
     const tokenOutAmount = ref('');
     const tradeSuccess = ref(false);
     const trading = ref(false);
-    const txHash = ref('');
     const modalTradePreviewIsOpen = ref(false);
 
+    // COMPUTED
+    const getTokens = (params = {}) =>
+      store.getters['registry/getTokens'](params);
+    const getConfig = () => store.getters['web3/getConfig']();
+    const tokens = computed(() => getTokens({ includeEther: true }));
```

Comment:
```suggestion
    const { allTokensIncludeEth: tokens } = useTokens();
```
This composable will be different after the registry is fully refactored and implemented but for now we have this legacy composable that works instead of using the getter.